dev/src/reference.ts

```diff
@@ -1458,6 +1458,33 @@ export class VectorQuerySnapshot<
   }
 }
 
+interface QueryStreamResponse<
+  AppModelType = firestore.DocumentData,
+  DbModelType extends firestore.DocumentData = firestore.DocumentData,
+> {
+  transaction?: Uint8Array;
+  readTime?: Timestamp;
+  explainMetrics?: ExplainMetrics;
+  document?: QueryDocumentSnapshot<AppModelType, DbModelType>;
+}
+
+interface QueryResponse<TSnapshot> {
+  transaction?: Uint8Array;
+  readTime?: Timestamp;
```

Comment:
Likewise, I think we can remove `readTime`, and only have it exist within the snapshot.

Reply:
Ok I agree, your comment from above does apply in this `QueryResponse` type. Will do.